time stamp

string solution_mode

bool attitude_valid
bool heading_valid
bool velocity_valid
bool position_valid
bool vert_ref_used
bool mag_ref_used
bool gps1_vel_used
bool gps1_pos_used
bool gps1_course_used
bool sol_align_valid
